object_detection/scripts/get_objects_and_coordinates_in_terminal.py

```diff
@@ -0,0 +1,141 @@
+"""
```

Comment:
Is this @SAlexisvf script? Or a new one?

Reply:
This is a new one based on the Alexis' one get_objects_and_coordinates.py